plenum/test/replica/test_replica_removing_with_backup_degraded.py

```diff
@@ -63,36 +46,55 @@ def txnPoolNodeSet(node_config_helper_class,
             txnPoolNodesLooper.removeProdable(node)
 
 
+def update_conf(tconf, tmp):
+    tconf.throughput_measurement_class = RevivalSpikeResistantEMAThroughputMeasurement
+    tconf.throughput_measurement_params = {
+        'window_size': 2,
+        'min_cnt': 3
+    }
+    tconf.ACC_MONITOR_TIMEOUT = 3
+    tconf.ACC_MONITOR_TXN_DELTA_K = 0
+    tconf.REPLICAS_REMOVING_WITH_DEGRADATION = tmp["replica_remove"]
+    tconf.ACC_MONITOR_ENABLED = tmp["monitor_strategy"]
+    tconf.PerfCheckFreq = 5
+    return tconf
+
+
 def test_replica_removing_with_backup_degraded(looper,
                                                txnPoolNodeSet,
                                                sdk_pool_handle,
                                                sdk_wallet_client,
+                                               sdk_wallet_steward,
                                                tconf,
                                                tdir,
                                                allPluginsPath):
     """
-    1. Start backup degraded.
-    2. Check that degraded replicas
-    3. Start View Change.
-    4. Check that all replicas were restored.
-    """
-    instance_to_remove = 1
-    view_no = txnPoolNodeSet[0].viewNo
+      Node will change view even though it does not find the master to be degraded
```

Comment:
Are we talking about a view change or removing of replicas here? Is the comment correct?